Selenium/Ex10.rb

```diff
@@ -0,0 +1,36 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = 'C:\Users\adity\Desktop\Drivers\chromedriver-win64\chromedriver-win64\chromedriver.exe'
+driver = Selenium::WebDriver.for :chrome
+
+
+driver.get 'https://computer-database.gatling.io/computers'
+
+
+column_header_text = "Computer name"
+
+
+header_row = driver.find_element(css: 'thead tr')
+
+
+column_index = header_row.find_elements(tag_name: 'th').find_index do |header_cell|
+  header_cell.text == column_header_text
+end
```

Comment:
You do not need to fetch the header row separately. You only need to read the `tr` elements of the table which will also fetch the header values. as they too are enclosed within a `tr` tag.